fetch.bs

```diff
@@ -4422,6 +4538,14 @@ steps:
 
  <li><p><a>Upgrade <var>request</var> to a potentially trustworthy URL, if appropriate</a>.
 
+ <li>
+  <p>Optionally, run <a>upgrade an HTTP request</a> algorithm on <var>request</var>, if appropriate</a>.
```

Comment:
Spurious trailing `</a>`.

Reply:
Also, "if appropriate" is rather unclear. Can't we just run this unconditionally and leave all the optionally and other implementation-defined determinations to "upgrade an HTTP request"?

Reply:
The "if appropriate" wording was from the mixed content upgrade (Step 5 of Main fetch).

I'm happy to remove the "Optionally" from here. The HTTPS-Upgrading section already starts with "User agents may optionally upgrade requests...". Do we need any additional wording there?

Reply:
I don't think so. The existing "if appropriate" are part of algorithm names. Potentially those algorithms should be renamed as well. But here the algorithm name doesn't contain it so we don't need it.